src/v2/view-builder/views/ov/ChallengeOktaVerifyTotpView.js

```diff
@@ -13,6 +15,38 @@ const Body = BaseForm.extend(Object.assign(
     save() {
       return loc('mfa.challenge.verify', 'login');
     },
+
+    initialize: function() {
+      BaseForm.prototype.initialize.apply(this, arguments);
+      this.listenTo(this.options.appState, 'afterError', (convertedErrors) => {
+        // Need to wait until BaseForm renders the error so we can replace it
+        setTimeout(() => {
+          this._showUserVerifcationError(convertedErrors);
+        });
+      });
+    },
+
+    _showUserVerifcationError(convertedErrors) {
+      const errorSummaryKeys = convertedErrors?.responseJSON?.errorSummaryKeys;
+      if (errorSummaryKeys && errorSummaryKeys.includes(OV_UV_ENABLE_BIOMETRIC_SERVER_KEY)) {
+        // remove the error generated by BaseForm
+        this.model.trigger('clearFormError');
+        // add custom error for User Verification
+        this.add('<div class="ion-messages-container"></div>', '.o-form-error-container');
+        const options = {
+          type: 'error',
+          className: 'okta-verify-uv-callout-content',
+          title: loc('oie.authenticator.app.method.push.verify.enable.biometrics.title', 'login'),
+          subtitle: loc('oie.authenticator.app.method.push.verify.enable.biometrics.description', 'login'),
+          bullets: [
+            loc('oie.authenticator.app.method.push.verify.enable.biometrics.point1', 'login'),
```

Comment:
I think we should avoid hardcoding these error messages. Widget should strictly respect what contains in API response, that was the purpose of `ION` framework.. If API is not sending this part of error message then we should ask backend team to come up with some solution so that it would be easy for widget to follow API structure and show this enhance error messaging.. 

It could be done this way
```
"messages": {
    "type": "array",
    "value": [
      {
        "message": "Your response was received, but your organization requires biometrics. Make sure your device supports biometrics, Okta Verify is up-to-date and biometrics are enabled for your account in Okta Verify, then try again.",
        "i18n": {
          "key": "oie.authenticator.oktaverify.method.totp.verify.enable.biometrics",
          "params": []
        },
        "class": "ERROR",
        "bullets": ['bullet1", "bullet2", "bullet3"]
      }
    ]
  }
 ```
IDX is doing a cleanup for `ResendView` that hardcodes all of warning messages.. remediation should send those messages.. https://oktainc.atlassian.net/browse/OKTA-408316

On similar note this can be done as well..

Reply:
we already have this pattern in another [OV view](https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/views/ov/ChallengeOktaVerifyResendPushView.js#L37). I agree we should stick with what the API returns, but this is also harder since what we are doing here is cosmetic changes to show a better error message (improving ux), and for a pure API client that is really not needed.
Probably we should talk to idx to see how we can improve the error message object that is returned. We can do a follow up for that, and if/when something changes, we can come back and adjust this code

Reply:
@mauriciocastillosilva-okta thanks Mau.. can we create a jira for this and assign to appropriate team so that we don't miss this cleanup.